tests/test_app_server.py

```diff
@@ -393,9 +428,13 @@ def test_unhandled_error(self):
             eq_(500, response.status_code)
             eq_("An internal error occured", response.data)
 
-        # Try it again with debug=True to get a stack trace instead of
-        # a generic error message.
-        handler = ErrorHandler(self.app, debug=True)
+
+    def test_unhandled_error_debug(self):
```

Comment:
The exception in test_unhandled_error rolls back the database session, and this test needs a working database session to set the log level, so I moved it to a separate test method.